hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -161,6 +161,8 @@ caption.maintenancewindow.schedule = Schedule
 caption.maintenancewindow.duration = Duration
 caption.maintenancewindow.timezone = Time Zone (from GMT)
 
+caption.file.upload.inprogress = File upload in progress
```

Comment:
caption of the dialog should indicate more clearly what is going to happen when clicking OK. 
e.g. "Stop file upload and leave page"
Or something shorter.

Reply:
DONE